galaxy.yml

```diff
@@ -0,0 +1,16 @@
+namespace: community
+name: mysql
+version: 1.0.0
+readme: README.md
+authors:
+  - MySQL Working Group (https://github.com/ansible/community/wiki/MySQL)
+description: MySQL collection for Ansible
+license_file: LICENSE
```

Comment:
If you want to name that file `LICENSE`, you need to adjust all copyright headers which refer to `COPYING`.

Reply:
thanks Felix! I've renamed the file from LICENSE to COPYING